app/__init__.py

```diff
@@ -1,3 +1,113 @@
-from flask import Flask
+from flask import Flask, request
+from flask_restful import Resource, Api, reqparse, abort
 
 app = Flask(__name__)
+api = Api(app)
+
+requests_store = []
+users = {}
+
+parser = reqparse.RequestParser()
+parser.add_argument('title', required=True, type=str)
+parser.add_argument('location', required=True, type=str)
+parser.add_argument('request_type', required=True)
+parser.add_argument('description')
+
+
+def find_request(request_id):
+    """ Find a specific request resource based off the id """
+    return [_request for _request in requests_store if _request['request_id'] == request_id]
+
+
+class RequestList(Resource):
+    def get(self):
```

Comment:
How do you ensure that users can only view their own requests, and the admin can view all the requests?

Reply:
Okay working on the two changes.